package.json

```diff
@@ -1,76 +1,94 @@
-	"main": "src/index.js",
-	"scripts": {
-		"test": "jest",
-		"lint": "eslint src",
-		"lint:fix": "eslint --fix src",
-		"prepare": "husky install"
-	},
-	"author": "ADAMANT Foundation <devs@adamant.im> (https://adamant.im)",
-	"license": "GPL-3.0",
-	"dependencies": {
-		"@liskhq/lisk-cryptography": "3.2.1",
-		"axios": "^1.5.0",
-		"bignumber.js": "^9.1.1",
-		"bip39": "^3.1.0",
-		"bitcoinjs-lib": "^6.1.3",
-		"bytebuffer": "^5.0.1",
-		"coininfo": "^5.2.1",
-		"ecpair": "^2.1.0",
-		"ed2curve": "^0.3.0",
-		"ethereumjs-util": "^7.1.5",
-		"hdkey": "^2.1.0",
-		"pbkdf2": "^3.1.2",
-		"socket.io-client": "^4.7.2",
-		"sodium-browserify-tweetnacl": "^0.2.6",
-		"tiny-secp256k1": "^2.2.3"
-	},
-	"publishConfig": {
-		"access": "public"
-	},
-	"keywords": [
-		"adm",
-		"adamant",
-		"blockchain",
-		"messenger",
-		"chat",
-		"decentralized messenger",
-		"anonymous messenger",
-		"secure messenger",
-		"wallet",
-		"crypto wallet",
-		"private keys",
-		"communication",
-		"decentralized",
-		"decentralization",
-		"anonymous",
-		"anonymity",
-		"secure",
-		"encrypted",
-		"encryption",
-		"crypto",
-		"cryptocurrency",
-		"bitcoin",
-		"ethereum"
-	],
-	"repository": {
-		"type": "git",
-		"url": "git+https://github.com/Adamant-im/adamant-api-jsclient.git"
-	},
-	"bugs": {
-		"url": "https://github.com/Adamant-im/adamant-api-jsclient/issues"
-	},
-	"homepage": "https://github.com/Adamant-im/adamant-api-jsclient#readme",
-	"devDependencies": {
-		"@commitlint/cli": "^17.7.1",
-		"@commitlint/config-conventional": "^17.7.0",
-		"eslint": "^8.48.0",
-		"eslint-config-google": "^0.14.0",
-		"eslint-plugin-jest": "^27.2.3",
-		"husky": "^8.0.3",
-		"jest": "^29.6.4"
-	}
+  "name": "adamant-api",
+  "version": "1.8.0",
+  "description": "REST API for ADAMANT Blockchain",
+  "keywords": [
+    "adm",
+    "adamant",
+    "blockchain",
+    "messenger",
+    "chat",
+    "decentralized messenger",
+    "anonymous messenger",
+    "secure messenger",
+    "wallet",
+    "crypto wallet",
+    "private keys",
+    "communication",
+    "decentralized",
+    "decentralization",
+    "anonymous",
+    "anonymity",
+    "secure",
+    "encrypted",
+    "encryption",
+    "crypto",
+    "cryptocurrency",
+    "bitcoin",
+    "ethereum"
+  ],
+  "homepage": "https://github.com/Adamant-im/adamant-api-jsclient#readme",
+  "bugs": {
+    "url": "https://github.com/Adamant-im/adamant-api-jsclient/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Adamant-im/adamant-api-jsclient.git"
+  },
+  "license": "GPL-3.0",
+  "author": "ADAMANT Foundation <devs@adamant.im> (https://adamant.im)",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "jest",
+    "lint": "gts lint",
+    "preinstall": "npx only-allow pnpm",
+    "prepare": "husky install",
+    "clean": "gts clean",
+    "compile": "tsc",
+    "fix": "gts fix",
+    "pretest": "npm run compile",
+    "posttest": "npm run lint"
+  },
+  "dependencies": {
```

Comment:
We can update to the latest versions:

```
ncu
Checking /package.json
[====================] 22/22 100%

 @commitlint/cli            ^17.7.1  →  ^17.7.2
 @liskhq/lisk-cryptography    3.2.1  →    3.3.0
 axios                       ^1.5.0  →   ^1.5.1
 bignumber.js                ^9.1.1  →   ^9.1.2
 bitcoinjs-lib               ^6.1.3  →   ^6.1.5
 eslint                     ^8.48.0  →  ^8.51.0
 eslint-plugin-jest         ^27.2.3  →  ^27.4.2
 jest                       ^29.6.4  →  ^29.7.0
```

We can also address `1 moderate severity vulnerability` (`semver  <5.7.2`) with `npm audit fix`